.github/workflows/CI-audit.yml

```diff
@@ -0,0 +1,26 @@
+name: Audit
+
+run-name: "Workflow CI/CD Steps: cargo audit"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: zencash/sc-ci-base
+  IMAGE_TAG: noble_rust-stable_latest
+  DOCKER_COMPOSE_CMD: "docker compose -f ${GITHUB_WORKSPACE}/ci/docker-compose.yml run --rm cargo-container"
+  DOCKER_BUILD_DIR: /build
+  DOCKER_CARGO_HOME: /tmp/.cargo
+  CARGO_BINARIES_INSTALL: "cargo-audit"
+
+jobs:
+  audit:
+    runs-on:
+      group: ${{ vars.RUNNER_GROUP }}
```

Comment:
Given that this workflow doesn't need to compile the nodes, I would consider using the lower-spec runners `ubuntu-latest`.